public/js/btn_back_to_top.js

```diff
@@ -0,0 +1,14 @@
+window.onscroll = function() {scrollFunction()};
```

Comment:
Tentamos retirar a chamada de function adicional e sem isto o botão back to top desaparecia e apenas voltava a aparecer após dar um f5